packages/office-ui-fabric-react/src/components/Callout/CalloutContent.tsx

```diff
@@ -76,7 +76,8 @@ export class CalloutContent extends BaseComponent<ICalloutProps, ICalloutState>
   }
 
   public componentWillUpdate(newProps: ICalloutProps) {
-    if (newProps.targetElement !== this.props.targetElement || newProps.target !== this.props.target) {
+    // If the target element changed, find the new one. If we are tracking target with class name, always find element because we do not know if fabric has rendered a new element and disposed the old element.
+    if (newProps.targetElement !== this.props.targetElement || newProps.targetElement == null) {
```

Comment:
We still need to check `newProps.target ` targetElement is deprecated and is really only there until we remove it in a major update. We really don't care if targetElement is null generally. Actually I'm not sure I understand this change. 

I made this callout codepenand it looks like it's fine if the state changes.

https://codepen.io/slimemold/pen/PjpEzv?editors=0010